pkg/deploy/devfile_registry.go

```diff
@@ -27,49 +27,90 @@ type DevFileRegistryConfigMap struct {
 }
 
 const (
-	DevfileRegistry = "devfile-registry"
+	DevfileRegistry              = "devfile-registry"
+	devfileRegistryGatewayConfig = "devfile-registry-gtw"
```

Comment:
I've named these like `che-gateway-route-devfile-registry` in helm. I don't like `gtw` in your naming. Otherwise I'm not suggesting to change. Might be good to sync operator and helm. wdyt ?